tools/wptserve/wptserve/server.py

```diff
@@ -839,7 +843,7 @@ def __init__(self, host="127.0.0.1", port=8000,
             self.started = False
 
             _host, self.port = self.httpd.socket.getsockname()
-        except Exception:
+        except Exception as e:
```

Comment:
unit test failed due to this line:

./wptserve/wptserve/server.py:846:9: F841 local variable 'e' is assigned to but never used

Reply:
Thanks, fixed now